### ISSUE TITLE
UDR extension sql for create bdr_internal_sequence_reset_cache error

### ISSUE THREAD

**Issue by digoal** [2ndQuadrant/bdr#132]
```
# cd /opt/pgsql/share/extension
[root@digoal extension]# cat bdr.control |grep default_version
default_version = '0.9.2.0'

# vi bdr--0.9.2.0.sql 
-- must comment this function , or it will error.
-- CREATE OR REPLACE FUNCTION bdr.bdr_internal_sequence_reset_cache(seq regclass)
-- RETURNS void LANGUAGE c AS 'MODULE_PATHNAME' STRICT;
```

**Comment by ringerc** [comment]
(Please format your posts with markdown, per the help)

This is a known issue with UDR version 0.9.2. It got missed due to the focus on BDR. A fix is pending with 0.9.3, which will be released in a few days now.

**Comment by ringerc** [comment]
Fixed in 0.9.3
